.github/pull_request_template.md

```diff
@@ -1,42 +1,60 @@
+    🌻 Thank you for your contribution to the polyPod repo.
+    
+    Please use the content below as a template for your pull request.
 
-    Please, make sure that:
-    - You wait till the build is finished and is green
-    - Tests for the changes have been added, if possible (for bug fixes / new features)
-    - Docs have been reviewed and added / updated, if needed (for bug fixes / new features)
-    - Commits in this PR are minimal and have descriptive commit messages.
+    Make sure that:
+    - You wait till the build is finished and is green 🟢
+    - Tests for the changes have been added, if possible (for bug fixes / new features) 🧪
+    - Docs have been reviewed and added / updated, if needed (for bug fixes / new features) 📄
+    - Commits in this PR are minimal and have descriptive commit messages. 💌
 
-    Before submitting this PR, please fill the following information on the following sections.
+    Feel free to remove sections which do not make sense. 🗑️
 -->
 
+## Scope
+<!-- Brief description of WHAT you’re doing and WHY. -->
 
-# ✍️ Description
-<!-- 
-    Please, include a summary of the changes you made here and the related issue.
-    Some questions to help you give more context:
-     - What is the current behavior?
-     - Why we need this change/addition?
-     - What is the new behavior (if this a feature change)?
-     - What should we expect after this change/addition? 
-     - How did you test it?
-     - Does this introduce a breaking change?
-     - Are there any dependencies that are required for this change?
- -->
-
-<!-- Link here the related JIRA issue in case of a bug / feature -->
-### 🏗️ Fixes (#jira-issue-number or link)
-
-
-<!-- Optional -->
-## ℹ️ Other information
+#### Closes [PROD4POD-###](https://jira.polypoly.eu/browse/PROD4POD-###)
+
+
+## 🏗️ Implementation 
+<!--
+    Some description of HOW you achieved it. 
+    Perhaps give a high level description of the program flow. 
+    Did you need to refactor something? 
+    What trade-offs did you take? 
+    Are there things in here which you’d particularly like people to pay close attention to?
+-->
+
+#### Paired with @pair
+
+
+## 🧪 How to Test 
+<!--
+    A straightforward scenario of how to test your changes could help colleagues that are not familiar with the part of the code that you are changing, but want to see it in action. 
+    This section can include a description or step-by-step instructions of how to get to the state of v2 that your change affects.
+
+    A "How To Test" section can look something like this:
+
+    - Open the app 
+    - Open `show_awesome_elephant_gifs` feature 
+    - You should see a GIF with elephants dancing
+-->
+
+
+## 📷 Screenshots 
 <!-- 
-    Any other information that is important to this PR. 
-    e.g. if regards a frontend UI change, add some screenshots 📸
-    of how the app looked before and how it looks after the change.
+    If regards UI changes, provide screenshots 📸 of how the app looked
```

Comment:
alt: implies instead of regards
Also, probably optional.

Reply:
well, thought the `If regards UI changes` points it out that is optional, no? :p